provider/kv.go

```diff
@@ -162,6 +162,9 @@ func (provider *Kv) list(keys ...string) []string {
 func (provider *Kv) listServers(backend string) []string {
 	serverNames := provider.list(backend, "/servers/")
 	return fun.Filter(func(serverName string) bool {
+		if _, err := provider.kvclient.Get(fmt.Sprint(serverName, "/url")); err != nil {
```

Comment:
First time getting in touch with our k/v store interfaces and libkv in general, so behold. :-)

Looking at https://godoc.org/github.com/docker/libkv/store#pkg-variables, it seems like we should be able to distinguish between `ErrKeyNotFound` and other errors. If that's feasible, I think we should at least log unexpected errors.

WDYT?